WebApiCoreSeed.Data/EF/DatabaseSeed.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace WebApiCoreSeed.Data.EF
+{
+    using System;
+    using System.Linq;
+    using WebApiCoreSeed.Data.Models;
+
+    /// <summary>
+    /// Helper class to seed sample data into a <see cref="WebApiCoreSeedContext"/>
+    /// </summary>
+    public static class DatabaseSeed
+    {
+        /// <summary>
+        /// Initializes a <see cref="WebApiCoreSeedContext"/> with sample Data
+        /// </summary>
+        /// <param name="dbContext">Context to be initialized with sample data</param>
+        public static void Initialize(WebApiCoreSeedContext dbContext)
+        {
+            // The following line is commented as it'll mess with the EF tracking table. We will always create the Data Structure using the migrations
+            // dbContext.Database.EnsureCreated();
+
+            if (!dbContext.Users.Any())
+            { 
+                var users = new[]
+                {
+                    new User { CreatedBy = "System", CreatedOn = DateTime.Now, Email = "noreply@makingsense.com", FirstName = "John", LastName = "Doe", UserName = "JohnDoe" },

```

Comment:
I don't think it's important, but this is what I was thinking: if this is meant to "mock" the data that would come out of the database, an average object from the database would likely have all the properties populated. I saw that the database model had a bunch of more properties that were not in here, so I just thought it would be a good idea to include them.

But again, it is not important at all. I'm actually fine with this.